app/views/create_new_user_requests/_requested_user_details.html.erb

```diff
@@ -28,7 +28,7 @@
       <% end %>
     </span>
     <span class="form-wrapper">
-      <%= r.select :organisation, options_for_select(@organisations_list), { include_blank: true }, multiple: false, class: "select2 form-control", type: "organisation" %>
```

Comment:
Could you please mention/justify the removal of `select2` class, which I believe will remove the option to search for a department in the list.

Reply:
Oops! Thanks for spotting it - I removed it while debugging and forgot to re-add it! 😬

Reply:
Addressed in https://github.com/alphagov/support/pull/1248/commits/0cd495b655483865b0abca4631caed58b7f5c466